src/main/java/com/conveyal/r5/util/ShapefileReader.java

```diff
@@ -77,6 +82,14 @@ public ReferencedEnvelope getBounds () throws IOException {
         return source.getBounds();
     }
 
+    public double getAreaSqKm () throws IOException, TransformException, FactoryException {
+        CoordinateReferenceSystem webMercatorCRS = CRS.decode("EPSG:3857");
+        MathTransform webMercatorTransform = CRS.findMathTransform(crs, webMercatorCRS, true);
+        Envelope mercatorEnvelope = JTS.transform(getBounds(), webMercatorTransform);
```

Comment:
This is going to grossly inflate the size in m2 of bounding boxes at high latitudes right? I guess we can tolerate that if the upper limit on size is permissive enough.